Makefile

```diff
@@ -99,6 +100,26 @@ $(addprefix $(DEST)/, $(MAIN_TARGET)): $(DEST)/% :
 	stg repair
 	stg import -s ../patch/series
 
+	mkdir $(NON_UP_LOC)
```

Comment:
rm -rf $(NON_UP_LOC) before this line to start from scratch

Reply:
`rm -rf $(BUILD_DIR)` is removed during the start of the build, NON_UP_LOC is present under BUILD_DIR, so this isn't required